### ISSUE TITLE
display value transferring results in the execution

### ISSUE THREAD

**Issue by puncsky** [iotexproject/iotex-explorer#996]
example: https://etherscan.io/tx/0xfbda161100f78f4699c2dda490a87cb1df77c9ecd8fc4a8858127e5efb62e359

![141568092515_ pic_hd](https://user-images.githubusercontent.com/1387285/64639992-e8d40700-d3bd-11e9-97d5-94ac98490e94.jpg)

Backend change: https://github.com/iotexproject/iotex-analytics/pull/86

```
type Action {
    byDates(startDate: Int!, endDate: Int!): ActionList
    evmTransfers(actHash: String!): [EvmTransfer]!
}

type EvmTransfer{
    from: String!
    to: String!
    quantity: String!
}
```

**Comment by Frankonly** [comment]
Latest design
```
type Action {
    byDates(startDate: Int!, endDate: Int!): ActionList
    byHash(actHash: String!): ActionDetail
}

type ActionDetail{
    actionInfo: ActionInfo
    evmTransfers: [EvmTransfer]!
}

type EvmTransfer{
    from: String!
    to: String!
    quantity: String!
}
```

**Comment by nmhung1210** [comment]
> Latest design
> 
> ```
> type Action {
>     byDates(startDate: Int!, endDate: Int!): ActionList
>     byHash(actHash: String!): ActionDetail
> }
> 
> type ActionDetail{
>     actionInfo: ActionInfo
>     evmTransfers: [EvmTransfer]!
> }
> 
> type EvmTransfer{
>     from: String!
>     to: String!
>     quantity: String!
> }
> ```

It seems were not deployed yet. when will it be ready?

**Comment by lizhefeng** [comment]
We are rebuilding the table. Will let you know when it is ready. @nmhung1210

**Comment by lizhefeng** [comment]
@nmhung1210 It is ready. You can try it now.

**Comment by nmhung1210** [comment]
@lizhefeng what's unit for the `quantity`? Is it always `IOTX` or might be something else?

**Comment by Frankonly** [comment]
> @lizhefeng what's unit for the `quantity`? Is it always `IOTX` or might be something else?

The unit is `Rau` actually. 1 `IOTX` = 10^18 `Rau`. Refer to https://github.com/iotexproject/iotex-core/blob/master/pkg/unit/unit.go

**Comment by nmhung1210** [comment]
> > @lizhefeng what's unit for the `quantity`? Is it always `IOTX` or might be something else?
> 
> The unit is `Rau` actually. 1 `IOTX` = 10^18 `Rau`. Refer to https://github.com/iotexproject/iotex-core/blob/master/pkg/unit/unit.go

I'm confusing about this API now. For an example of xrc20 transfer execution at https://iotexscan.io/action/b4342d17aaff54a540533a56fa31d69c3c9315c77891c706e167c4041f633b22. It was a VITA transfer (for 1 VITA). And the response did not contain the VITA transfer value as below:
```
{
  "data": {
    "action": {
      "byHash": {
        "evmTransfers": [
          {
            "from": "io1eaeqwg2qd2y5e3647daah7qlmkvxupwzw2jq4w",
            "to": "io1hp6y4eqr90j7tmul4w2wa8pm7wx462hq0mg4tw",
            "quantity": "0"
          }
        ]
      }
    }
  }
}
```
Could you please help to check if we can add token's `value` to the type `EvmTransfer` type?

**Comment by lizhefeng** [comment]
@nmhung1210 The evmTransfer is for native transfer tracking only. But one thing we can do is to integrate Xrc20ByHash here as well.

**Comment by lizhefeng** [comment]
@nmhung1210 Can we make native token tracking work first?

**Comment by nmhung1210** [comment]
> @nmhung1210 Can we make native token tracking work first?

ok, I'll make it for native token first.

**Comment by nmhung1210** [comment]
Closed in favor of #1028
